content/master/concepts/packages.md

```diff
@@ -177,7 +177,7 @@ provided constraints.
 > Dependency resolution is a `beta` feature and depends on the `v1beta1`
 > [`Lock` API][lock-api].
 
-For an example Configuration package, see [getting-started-with-gcp].
+For an example Configuration package, see [getting-started-with-gcp](https://github.com/crossplane/docs/tree/master/content/v1.10/snippets/package/gcp).
```

Comment:
Points to v1.10 until this chapter is revamped.

Reply:
Do we need to switch the link style to make this work? Just trying to be consistent within a single doc

Reply:
Not required but slowly migrating to inline links as they are changed. I didn't want to do a full rewrite of all links in this PR.

Reply:
ack